### PR TITLE
Fixed warnings

### PR DESCRIPTION
**Pull request by Nabile-Rahmani**
Only ~169 to go.

### CODE REVIEW

File: osu.Framework/Graphics/Containers/Container.cs [hunk 1]
```diff
@@ -508,7 +506,7 @@ private static void addFromContainer(int treeIndex, ref int j, Container<T> pare
             {
                 Drawable drawable = current[i];
 
-                while (drawable != (drawable = drawable.Original)) ;
+                while (drawable != (drawable = drawable.Original)) { }
```
Comment (peppy) [inline osu.Framework/Graphics/Containers/Container.cs]: was this source of a warning?

Comment (Nabile-Rahmani) [inline osu.Framework/Graphics/Containers/Container.cs]: On Mono, yes.

File: osu.Framework/IO/Stores/OnlineStore.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@ public class OnlineStore : IResourceStore<byte[]>
         {
             return await Task.Run(delegate
             {
-                if (!url.StartsWith(@"https://"))
+                if (!url.StartsWith(@"https://", System.StringComparison.Ordinal))
```
Comment (peppy) [inline osu.Framework/IO/Stores/OnlineStore.cs]: use `using System;` in this file plz

Comment (Nabile-Rahmani) [inline osu.Framework/IO/Stores/OnlineStore.cs]: Roger.

**Pull request merged**